epsagon/trace.py

```diff
@@ -461,6 +468,7 @@ def send_traces(self):
             return
         trace = ''
         try:
+            trace_factory.remove_trace()
```

Comment:
It's OK to do it in the beginning, but I think that it's better to do it in `finally` block - if some code will be added before this line it will be problematic (it could fail and the trace will not be removed)

Reply:
K :)